### ISSUE TITLE
Can we delete this repo?

### ISSUE THREAD

**Issue by willwhite** [mapbox/to-fix#1]
@srividyacb @batpad what is this repo used for? Can we delete it?

**Comment by batpad** [comment]
Yes. @willwhite seems like it requires admin permissions to delete. Are you able to delete this? If so, please go ahead, thank you.

**Comment by michaelsteffen** [comment]
@willwhite can you please delete this repo? You seem to be the only one with admin priviledges here.
